### PR TITLE
Building Dungeon 3

### PR DESCRIPTION
**Pull request by AlexanderGregorSanchez**

### CODE REVIEW

**Comment by AlexanderGregorSanchez** [comment]
forget to add to [d7857a8](https://github.com/KoolieAid/CookingYnah/pull/24/commits/d7857a8fbda5f7dd0d32433b9886504be45f37f5) but i also change Dungeon Level name to Dungeon 1 Level

File: Assets/Scripts/Managers & Handlers/FireLightHandler.cs
Comment (LauSnchzz) [inline Assets/Scripts/Managers & Handlers/FireLightHandler.cs:1]: idk if this is taxing to the device or not pero if its okay then its okay HAHA

**Review by LauSnchzz — APPROVED** [review]
all goods

File: Assets/Scripts/Managers & Handlers/FireLightHandler.cs
Comment (LauSnchzz) [reply Assets/Scripts/Managers & Handlers/FireLightHandler.cs:1]: ikaw na lng mag resolve alex if you just want to leave it be

Comment (AlexanderGregorSanchez) [reply Assets/Scripts/Managers & Handlers/FireLightHandler.cs:1]: There is nothing else going on in that scene so it would be fine

**Pull request merged**